cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationTest.java

```diff
@@ -212,4 +212,16 @@ public void searchItemByHoldingsPermanentLocationId() throws Exception {
         + "WHERE permanentLocationId='53cf956f-c1df-410b-8bea-27f712cca7c0')";
     assertEquals(expected, sql);
   }
+
+
+    @Test
+  public void searchItemByHoldingsPermanentLocationName() throws Exception {
```

Comment:
The method name is misleading. What about `fieldNameContainsDot`?

Reply:
done